kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -366,17 +366,17 @@ public String getClusters()
     return Joiner.on(';').join(clusters);
   }
 
-  public void setTopics(String... topics)
+  public void setTopics(String topics)
   {
-    this.topics = topics;
+    this.topics = topics.split(",");
```

Comment:
Please add javadoc that mentions that `,` is used for splitting. 
Does it make sense to let user define this separator?
Is this not an API change?